llm_bench/python/utils/memory_profile.py

```diff
@@ -23,15 +25,29 @@ def collect_memory_consumption(self):
             self.g_event.wait()
             while True:
                 process = psutil.Process(os.getpid())
-                rss_mem_data = process.memory_info().rss / float(2**20)
                 try:
-                    shared_mem_data = process.memory_info().shared / float(2**20)
```

Comment:
Can move "/ float(2**20)" to get_max_memory_consumption, so that we can save a little time.